### ISSUE TITLE
Some Icon colors forced white

### ISSUE THREAD

**Issue by codeitco** [jccguimaraes/atom-project-viewer#102]
# ISSUE
Some Icons are forced white, regardless of the assigned group color.

## Environment
Windows 10
ATOM 1.12.6

## VERIFY
When adding/updating a project or group, there are some icons (such as the alignment icons) that appear bright white and don't adhere to the group color that is assigned.

**Comment by jccguimaraes** [comment]
can you share a small preview of this please?

**Comment by codeitco** [comment]
![icon-color-bug-1](https://cloud.githubusercontent.com/assets/20772638/20974337/d80b0342-bc69-11e6-8f1f-f2defc5b9faa.jpg)
![icon-color-bug-2](https://cloud.githubusercontent.com/assets/20772638/20974336/d8054218-bc69-11e6-8862-a4ff20b79296.jpg)

**Comment by jccguimaraes** [comment]
hey @codeitco!
I can't replicate this issue :(

**Comment by codeitco** [comment]
Its weird eh!? So in your icon layout screen as in my 2nd screenshot, do you see the icons as dim grey or whiter? I thought maybe something in my styles.less file is messing it up but I don't see anything unusual.

/*
 * Your Stylesheet
 *
 * This stylesheet is loaded when Atom starts up and is reloaded automatically
 * when it is changed and saved.
 *
 * Add your own CSS or Less to fully customize Atom.
 * If you are unfamiliar with Less, you can read more about it here:
 * http://lesscss.org
 */


/*
 * Examples
 * (To see them, uncomment and save)
 */

// style the background color of the tree view
.tree-view {
  // background-color: whitesmoke;
}

// style the background and foreground colors on the atom-text-editor-element itself
atom-text-editor {
  // color: white;
  // background-color: hsl(180, 24%, 12%);
}

// To style other content in the text editor's shadow DOM, use the ::shadow expression
atom-text-editor::shadow .cursor {
  // border-color: red;
}

// modify the spacer object of the toolbar
.tool-bar.tool-bar-horizontal .tool-bar-spacer {
    border: 0 none;
    border-left: 1px solid #5dc0ff;
    border-right: 1px solid #111314;
    -webkit-mask-image: linear-gradient(to bottom, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.7), #000000, rgba(0, 0, 0, 0.7), rgba(0, 0, 0, 0));
    margin: 5px 20px;
}

// Move atom notifications to bottom right
atom-notifications {
  top: auto;
  bottom: 50px;
}

// Modify Colours in Nav-Panel extension
.zi-marker-panel .list-item .icon:hover:before {
  color: orange;
  text-shadow: 0px 0px 8px red;
}
.zi-marker-panel .list-item.zi-highlight .icon:hover:before {
  color: black;
  text-shadow: 0px 0px 8px gray;
}
.zi-marker-panel .list-item:hover {
	background-color: @background-color-highlight
}
.zi-marker-group-label>span {
	font-size: 14px;
	font-weight: bold;
}
.zi-marker-panel .zi-highlight>span {
  color: orange;
}
atom-text-editor div.line-number.zi-highlight,
atom-text-editor::shadow div.line-number.zi-highlight {
  background-color: #5e5e5e;
  color: #ffffff;
}

// Adjust quick editor to show panel properly
.quick-editor {
  overflow: hidden;

  atom-text-editor {
    position: absolute;
    width: 100%;
    height: 100%;
  }
}

// Adjust Indent Guide
.indent-guide-improved {
  &.indent-guide-stack {
    background: repeating-linear-gradient(to bottom, rgba(158, 158, 158, 0.2), rgba(0, 0, 0, 0) 2px );

    &.indent-guide-active {
      background: repeating-linear-gradient(to bottom, rgba(255, 255, 255, 0.3), rgba(0, 0, 0, 0) 2px );
    }
  }
  background: repeating-linear-gradient(to bottom, rgba(158, 158, 158, 0.2), rgba(0, 0, 0, 0) 2px );
}


// Resize UI
html, body { font-size: 13px; }

**Comment by codeitco** [comment]
Btw, there are only 6 or 7 icons doing this, so maybe not worth fixing anyways. If I ever discover anything new about this issue, I'll be sure to report it.

On the other hand, if you need more info from me regarding other packages, atom settings etc, I'll be happy to provide :)

Thanks for your time and consideration on this.

**Comment by jccguimaraes** [comment]
try to comment this part

```
// Modify Colours in Nav-Panel extension
.zi-marker-panel .list-item .icon:hover:before {
color: orange;
text-shadow: 0px 0px 8px red;
}
.zi-marker-panel .list-item.zi-highlight .icon:hover:before {
color: black;
text-shadow: 0px 0px 8px gray;
}
.zi-marker-panel .list-item:hover {
background-color: @background-color-highlight
}
.zi-marker-group-label>span {
font-size: 14px;
font-weight: bold;
}
.zi-marker-panel .zi-highlight>span {
color: orange;
```

**Comment by jccguimaraes** [comment]
try to open the devTools and inspect the `.icon span::before` on those icons and see if anything is overriding it

**Comment by jccguimaraes** [comment]
i will close this issue and if you find the cause for it, we open it again ok?

**Comment by egbradshaw** [comment]
@jccguimaraes 

Thanks for the suggestion, I was able to drill down and find whats causing it.

On the element:
<span class="icon icon-file-text">
	::before == $0
NOTES / COMMENTS
</span>

The before that is there has this applied to it in dev tools

body:not(.seti-icons-no-color) atom-workspace .icon-file-binary:before, body:not(.seti-icons-no-color) atom-workspace .icon-file-pdf:before, body:not(.seti-icons-no-color) atom-workspace .icon-file-text:before, body:not(.seti-icons-no-color) atom-workspace .icon-book:before, body:not(.seti-icons-no-color) atom-workspace .icon-file-media:before, body:not(.seti-icons-no-color) atom-workspace .icon-markdown:before, body:not(.seti-icons-no-color) atom-workspace .title:before {
    color: #d4d7d6;
}

When I uncheck that color setting, the icon beside the text changes to the applied color through the pv group.

It clearly shows seti-icons being a consistent class name so I checked and I have the seti-icon package that is overwriting some of your icons. As soon as I disabled the seti-icons package, the icon in PV changed back to the default and color as well.

Are you able to tell who controls this conflict in this case?
